src/ray/core_worker/actor_manager.h

```diff
@@ -153,6 +166,14 @@ class ActorManager {
   /// the corresponding handles are gone out of scope.
   absl::flat_hash_map<ActorID, std::function<void(const ActorID &)>>
       actor_out_of_scope_callbacks_ GUARDED_BY(mutex_);
+
+  /// List of actor ids that didn't resolve its location in GCS yet.
+  /// This means that these actor information hasn't been persisted to GCS.
+  /// It happens only when the actor is not created yet because local dependencies
+  /// for actor creation task hasn't been resolved.
```

Comment:
```suggestion
  /// It happens only when the actor is not created yet because the owner hasn't resolved the dependencies
  /// for the actor creation task.
```

Suggestion to specify that it is the owner that needs to resolve the dependencies.